examples/pdf417.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const escpos = require('../');
+const iconv = require('iconv-lite');
+
+const device = new escpos.USB();
+
+const printer = new escpos.Printer(device);
+
+let data = 'Lorem Ipsum es simplemente el texto de relleno de las imprentas y archivos de texto. Lorem Ipsum ha sido el texto de relleno estándar de las industrias desde el año 1500, cuando un impresor (N. del T. persona que se dedica a la imprenta) desconocido usó una galería de textos y los mezcló de tal manera que logró hacer un libro de textos especimen. No sólo sobrevivió 500 años, sino que tambien ingresó como texto de relleno en documentos electrónicos, quedando esencialmente igual al original. Fue popularizado en los 60s con la creación de las hojas "Letraset", las cuales contenian pasajes de Lorem Ipsum, y más recientemente con software de autoedición, como por ejemplo Aldus PageMaker, el cual incluye versiones de Lorem Ipsum.';
+
+
+device.open(function () {
+
+    printer
+        .align('rt')
+        .pdf417(data)
```

Comment:
I wonder if I need to create a new `Printer.pdf417()` method even though the `Printer.barcode()` method already exists in escpos.